bitnami/redis/values.yaml

```
@@ -950,9 +950,12 @@ networkPolicy:
 ## ref: https://kubernetes.io/docs/concepts/policy/pod-security-policy/
 ##
 podSecurityPolicy:
-  ## @param podSecurityPolicy.create Specifies whether a PodSecurityPolicy should be created
+  ## @param podSecurityPolicy.create Specifies whether a PodSecurityPolicy should be created. You also need to set podSecurityPolicy.enabled to true
```

Let's try to keep the des
```suggestion
  ## @param podSecurityPolicy.create Specifies whether a PodSecurityPolicy should be created (set `podSecurityPolicy.enabled` to `true` to enable it)
```